src/moin/converter/_tests/test_moinwiki_in_out.py

```diff
@@ -19,8 +19,7 @@
 from . import serialize
 
 from moin.util.tree import moin_page, xlink, xinclude, html
-# from moin.converter.moinwiki_in import Converter as conv_in
-from moin.converter.moinwiki19_in import ConverterFormat19 as conv_in
+from moin.converter.moinwiki_in import Converter as conv_in
 from moin.converter.moinwiki_out import Converter as conv_out
```

Comment:
hmm, was the intention of this test to test the 1.9 -> 2.0 conversion?

is `moinwiki19_in` and `moinwiki_out` the stuff we use for upgrade?

Reply:
There is a test_moinwiki19_in_20_out.py to test the 1.9 -> 2.0 conversion.

m import uses moinwiki19_in and moinwiki_out to convert the most recent wiki page to 2.0 format. The older pages are left in 1.9 format.